Explain the new directory structure with composer-installer 5 (#2279)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/DirectoryStructure/Index.rst

:file:`public/`
---------------

This folder contains all files that are publicly available. Your webserver's
web root **must** point here.

This folder contains the main entry script :file:`index.php` created by Composer
and might contain publicly available files like a :file:`robots.txt` and
files needed for the server configuration like a :file:`.htaccess`.

If required, this directory can be renamed by setting `extra > typo3/cms > web-dir`
in the composer.json, for example to :file:`web`:

..  code-block:: json

>     :caption: composer.json

Reviewer: ```suggestion
    :caption: composer.json

```

The snippet is not rendered, probably because of missing empty line after caption.

are not linked anymore directly to the extension folders but to the directory
:file:`_assets/`.

..  note::

>     In TYPO3 v12 using the `typo3/cms-composer-installers` in version

Reviewer: I would rephrase it, as it is installed automatically, the sentence now can be understood as the user has to/can do something.
Maybe: "TYPO3 v12 requires typo3/cms-composer-installers in version 5."

are not linked anymore directly to the extension folders but to the directory
:file:`_assets/`.

..  note::

>     In TYPO3 v12 using the `typo3/cms-composer-installers` in version
>     5 is mandatory. Therefore the publicly available files provided by

Author: ```suggestion
    TYPO3 v12 requires `typo3/cms-composer-installers` in version
    5. Therefore the publicly available files provided by
```

:file:`public/typo3temp/assets/`
""""""""""""""""""""""""""""""""

> Directory :file:`typo3temp/assets/` contains temporary files that should be

Reviewer: ```suggestion
The directory :file:`typo3temp/assets/` contains temporary files that should be
```
